### ISSUE TITLE
How to increase perfomance ? FPS dropped after big count elements (200+)

### ISSUE THREAD

**Issue by evdevru** [liabru/matter-js#1104]
Hi, how to increase FPS ? Have any features ? I have stacked circles and after ~200 elements FPS start dropping without return.

Demo: https://codesandbox.io/s/matter-js-forked-ev6hzh?file=/src/App.vue
Debug: 
![image](https://user-images.githubusercontent.com/76659477/160122803-4476b6db-4455-4380-8bcf-77719225b656.png)

**Comment by liabru** [comment]
Are you using Vue here? I'm told Vue watchers can cause general performance problems, see this comment https://github.com/liabru/matter-js/issues/1001#issuecomment-998888681 and let me know if that is the cause?

**Comment by evdevru** [comment]
> Are you using Vue here? I'm told Vue watchers can cause general performance problems, see the comments here [#1001 (comment)](https://github.com/liabru/matter-js/issues/1001#issuecomment-998888681) and let me know if that is the cause?

Problem not with Vue. I make my code in vanilla js and get same result, drops after 130 elements:
(Vanilla JS) https://codesandbox.io/s/autumn-brook-uqc0ll
(Vue JS) https://codesandbox.io/s/matter-js-forked-ev6hzh

My system is Ryzen 2700x & 1660 ti (if it matters)
Results:
https://i.imgur.com/3PKnSB9.png
https://i.imgur.com/LTTu0Dm.png

Thats problem only if im using sprite. Without him i get drops only after 800 elements:
![image](https://user-images.githubusercontent.com/76659477/160248573-2dc3910b-395c-4b92-8c40-99b0bdd33ee7.png)

May be have optimization ways ?

**Comment by evdevru** [comment]
In my case working solution it change randerer to pixi.js. Using pixi i get drops only after 1000 elements, and after recalculate mass it was returned to 50 FPS with, then after 300 elements has 40 FPS.

![image](https://user-images.githubusercontent.com/76659477/160250064-68d47966-0118-4582-99bc-9d328b3928d3.png)


Its good result, but i dont want using others libs :(

**Comment by evdevru** [comment]
Problem if using SVG texture for sprite. If using PNG texture - result is good

**Comment by liabru** [comment]
Ah thanks for confirming, then it looks like this is browser limitation and you'll need to probably draw the SVG to another canvas texture to cache it if I understand correctly - you can probably make a copy of `Matter.Render` and customise it for this at the cost of extra memory usage.

Since that is more of an advanced rendering feature it's a little out of scope of this project at the moment unfortunately though, so closing this one but thanks for looking into it as it's one I'll make note of.
